lms/djangoapps/courseware/courses.py

```diff
@@ -135,8 +136,12 @@ def course_image_url(course):
         else:
             url += '/images/course_image.jpg'
     else:
-        loc = StaticContent.compute_location(course.id, course.course_image)
-        url = StaticContent.serialize_asset_key_with_slash(loc)
+        try:
+            loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+            url = StaticContent.serialize_asset_key_with_slash(loc)
+        except InvalidKeyError:
+            # handler for split courses, if course_image is blank
```

Comment:
for old mongo courses if course.course_image is blank its just work fine thats why using exception to determine split courses.